File: datasets/IR-Plag/original-java-rest-kotlin/case-01/plagiarized/L6/05/HelloWorld.kt
package IR

internal object HelloWorld {
    var s: String? = null
    fun Cetak() {
        for (a in 5 downTo 1) println(s)
    }

    @JvmStatic
    fun main(args: Array<String>) {
        s = "Welcome to Java"
        // code to print "Welcome to Java"
        Cetak()


        //End of code
        // 25779F8829AB7A7650E85A4CC871E6AC Ganteng
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-01/plagiarized/L4/04/HelloWorld.ts
package IR;

class HelloWorld {
    static s: string;
    static Cetak(): void {
        console.log(this.s);
        console.log(this.s);
        console.log(this.s);
        console.log(this.s);
        console.log(this.s);
    }
    public static main(args: string[]): void {
        this.s = "Welcome to Java";
        this.Cetak();
        // 25779F8829AB7A7650E85A4CC871E6AC Ganteng
    }
}

